README.md

```diff
@@ -67,3 +67,23 @@ will ensure actions in this repo are always used at the same commit. To do this:
   with:
     some-input: some-value
 ```
+
+### Releasing a version of shared-workflows
+
+When working with `shared-workflows`, we need to make sure that we won't break backwards compatibility.
+For this reason we need to provide releasable actions so engineers can review all the incoming changes,
+have they set automated update mechanisms (e.g. `dependabot`, `renovate` etc).
```

Comment:
This part of the sentence is somehow wrong.

Reply:
Slightly changed - not sure where it was somehow wrong but could you have another look @zerok?